src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -99,236 +99,236 @@
-    <sys:String x:Key="Activate_Display_Internet_Act_Events">Display Internet activation failure events</sys:String>
-    <sys:String x:Key="Activate_Display_KMS_Events">Display KMS activation events</sys:String>
+    <sys:String x:Key="Activate_Display_Internet_Act_Events">Visualizza gli eventi di errore di attivazione Internet</sys:String>
+    <sys:String x:Key="Activate_Display_KMS_Events">Visualizza gli eventi di attivazione del KMS</sys:String>
     <sys:String x:Key="Activate_Clear_Activation">Cancella attivazione</sys:String>
     <sys:String x:Key="Activate_Install_Licenses">Installa le licenze</sys:String>
-    <sys:String x:Key="Activate_Install_Licenses.Description">Get licenses from installed Office and install them.</sys:String>
+    <sys:String x:Key="Activate_Install_Licenses.Description">Ottieni le licenze da Office installato e installale.</sys:String>
     <sys:String x:Key="Activate_Install_Licenses_Button">Installa</sys:String>
     <sys:String x:Key="Activate_Install_Other_Licenses">Installa altre licenze</sys:String>
-    <sys:String x:Key="Activate_Install_Other_Licenses.Description">Install other licenses from local disk.</sys:String>
+    <sys:String x:Key="Activate_Install_Other_Licenses.Description">Installa altre licenze dal disco locale.</sys:String>
     <sys:String x:Key="Activate_Install_Other_Licenses_Button">Installa</sys:String>
-    <sys:String x:Key="Activate_Uninstall_All_Licenses">Uninstall all licenses</sys:String>
-    <sys:String x:Key="Activate_Uninstall_All_Licenses.Description">Uninstall all installed licenses. You will need to install license after this operation.</sys:String>
-    <sys:String x:Key="Activate_Uninstall_All_Licenses_Button">Uninstall</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses">Disinstalla tutte le licenze</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses.Description">Disinstalla tutte le licenze installate. Sarà necessario installare una licenza dopo questa operazione.</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses_Button">Disinstalla</sys:String>
 
-    <sys:String x:Key="Activate_Keys_Management">Keys management</sys:String>
-    <sys:String x:Key="Activate_Keys_Management.Description">Install, uninstall keys or activate Office offline.</sys:String>
+    <sys:String x:Key="Activate_Keys_Management">Gestione delle chiavi</sys:String>
+    <sys:String x:Key="Activate_Keys_Management.Description">Installa, disinstalla le chiavi o attiva Office offline.</sys:String>
     <sys:String x:Key="Activate_Product_Key">Product key</sys:String>
-    <sys:String x:Key="Activate_Product_Key.Description">Install Office product key, product key format: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
-    <sys:String x:Key="Activate_Install_Product_Key">_Install key</sys:String>
-    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Uninstall all unactivated keys</sys:String>
```

Comment:
It's recommand to set the hotkey same as English, like `Disinstalla t_utte le chiavi non attivate`.